blog-posts/guardian/lambda-versions/article.md

```diff
@@ -0,0 +1,57 @@
+---
+published: false
+title: 'AWS Lambda Versions : Time to clean up!'
+cover_image: https://raw.githubusercontent.com/pchol22/kumo-articles/master/blog-posts/guardian/lambda-versions/assets/cover_image.png
+description: ''
+tags: serverless, lambda, quality, AWS
+series:
+canonical_url:
+---
+
+_This article is part of a series on [Guardian][guardian], an open-source, highly configurable, automated best-practice audit tool for AWS serverless architectures._
+
+# Let's keep track
+
+Lambda versioning is great and might come in handy, but do you really need to keep dozens of outdated copies of your functions' code ? Here are two reasons why you should get rid of them.
+
+## 1. Less is more
+
+It sure can help to have a backup version of your code to rollback to in case of an emergency, but the day it happens, you will feel a lot more conformable only having to chose between two or three documented, aliased versions than being overwhelmed by dozens of anonymous and forgotten ones.
+
+![Button meme](./assets/button_meme.png 'Button Meme')
+
+## 2. Beware of AWS Lambda quotas
+
+AWS Lambda enforces a regional [75GB limit][quotas] for all your uploaded packages. This threshold might seem hard to reach and harmless, but it takes into account every version of every lambda you have deployed: dozens of versions per function multiplied by a large bundle size (see our [previous article][guardian-bundle-size-article]) will take you to past this ceiling in no time.
+
+It is possible to raise this soft limit by contacting the support, but it's not a sustainable workaround, and you don't want to be stuck unable to upload when there's an important fix to deploy.
+
+# Easily monitor your lambdas versioning with Guardian 🛡️
+
+[Guardian][guardian] now offers a **new rule** preventing your lambdas versions counter from going crazy.
+
+![Rule in CI](./assets/rule_CI.png 'Rule in CI')
```

Comment:
I think this screenshot does not put Guardian forward. Do you have the same with all green?